scripts/check_frontend_test_coverage_test.py

```diff
@@ -91,45 +100,37 @@ def test_get_stanzas_from_lcov_file(self):
             self.assertEqual(stanzas[2].covered_lines, 5)
 
     def test_get_stanzas_from_lcov_file_file_name_exception(self):
-        self.lcov_items_list = (
-            'SF:\n'
-            'LF:10\n'
-            'LH:5\n'
-            'end_of_record\n'
-        )
+        self.lcov_items_list = 'SF:\nLF:10\nLH:5\nend_of_record\n'
```

Comment:
This seems less readable than the previous one, can this change be reverted?

Ditto below.

Reply:
Agreed, fixed.